docs/framework/react/guide/tanstack-start.md

```diff
@@ -0,0 +1,246 @@
+---
+id: tanstack-start
+title: TanStack Start
+---
+
+TanStack Start is a full-stack framework for building server-rendered React applications built on top of [TanStack Router](https://tanstack.com/router).
+
+To set up a TanStack Start project, you'll need to:
+
+1) Install the dependencies
+2) Add a configuration file
+3) Create required templating
+
+# Install Dependencies
+
+TanStack Start is powered by [Vinxi](https://vinxi.vercel.app/) and [TanStack Router](https://tanstack.com/router) and requires them as dependencies.
+
+To install them, run:
+
+```shell
+npm i @tanstack/start @tanstack/react-router @tanstack/router-vite-plugin vinxi
```

Comment:
I think we can omit the `@tanstack/router-vite-plugin` right? Since we set this is already a dependency in the Start config for vite.
@tannerlinsley

Reply:
If we do need to use the plugin we should use `@tanstack/router-plugin` instead of `@tanstack/router-vite-plugin`.

Reply:
+1, I did not need to install a Vite plugin to run a Start site